py3status/i3status.py

```diff
@@ -548,8 +565,13 @@ def run(self):
                                     self.last_prefix = prefix
                                     self.set_responses(json_list)
                         else:
-                            err = self.poller_err.readline()
                             code = i3status_pipe.poll()
+                            # If we are 'suspended' then we recieve this
+                            # signal from i3status and can safely ignore
+                            # it.
+                            if code == -SIGUSR2:
```

Comment:
That was an odd thing but is no longer needed as we do not send the SIGUSR2 to i3status now.  Maybe an i3status bug?